content/reference-transaction-format.md

```diff
@@ -12,27 +12,48 @@ There are several different types of transactions that perform different actions
 
 Transactions are only valid if signed, submitted, and accepted into a validated ledger version. There are many ways a transaction can fail.
 
-* [Signing and Sending Transactions](#signing-and-sending-transactions)
+* [Authorized Transactions](#authorizing-transactions)
 * [Reliable Transaction Submission](#reliable-transaction-submission)
 * [Transaction Results - How to find and interpret transaction results](#transaction-results)
 * [Full Transaction Response List - Complete table of all error codes](#full-transaction-response-list)
 
-## Signing and Sending Transactions ##
+## Authorizing Transactions ##
 
-Signing a transaction cryptographically proves that the person in charge of the account sending the transaction is authorized to do so. Only signed transactions can be submitted to the network and included in a validated ledger. A signed transaction is immutable: its contents cannot change, and the signature is not valid for any other transaction.
+In the decentralized Ripple Consensus Ledger, a digital signature proves that a transaction is authorized to perform a specific set of actions. Only signed transactions can be submitted to the network and included in a validated ledger. A signed transaction is immutable: its contents cannot change, and the signature is not valid for any other transaction.
 
-You can sign a transaction using a secret key: either the master secret, or a regular secret if the account has a regular key pair associated with it. (See [SetRegularKey](#setregularkey) for details.)
+A transaction can be authorized by any of the following types of signatures:
 
-Multi-signature transactions are [in development](https://wiki.ripple.com/Multisign).
+* A single signature from the master secret key that is mathematically associated with the sending address. You can disable or enable the master key using an [AccountSet transaction](#accountset).
+* A single signature that matches a regular key associated with the address. You can add, remove, or replace a regular key using a [SetRegularKey transaction](#setregularkey).
+* A [multi-signature](#multi-signing) that matches a list of signers owned by the address. You can add, remove, or replace a list of signers using a [SignerListSet transaction](#SignerListSet).
 
-Typically, you create a transaction in JSON format first. Here is an example of an unsigned Payment-type transaction in JSON:
+Any signature type can authorize any type of transaction, with the following exceptions:
+
+* Only the master key can [disable the master key](#accountset-flags).
+* Only the master key can [permanently give up the ability to freeze](concept-freeze.html#no-freeze).
+* You can never remove the last method of signing transactions from an address.
+
+## Signing and Submitting Transactions ##
+
+Sending a transaction to the Ripple Consensus Ledger involves several steps:
+
+1. Create an unsigned transaction in JSON format.
```

Comment:
Maybe a link to give the user an idea of what an unsigned transaction looks like? e.g. https://ripple.com/build/transactions/#common-fields